<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_cstm">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>埋点事件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ppl">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>用户属性事件</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="edit_info">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>411</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_multi_thread">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>多线程埋点</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
